docker/populate

```diff
@@ -47,7 +47,7 @@ else
     sudo docker exec -it osoc_postgres_container_local_dev psql -U postgres osoc -c "UPDATE account SET role=0;" > /dev/null
     printf "created\n"
     printf "Adding other data: "
-    python3 ../docker/subpopulate.py
+    python3 ../docker/subpopulate.py $1
```

Comment:
$1 is wsl, this should be $2

Reply:
No because this is the path when the wsl flag is not given, so there is only 1 argument